projects/meta_cl/experiments/dendrites.py

```diff
@@ -0,0 +1,298 @@
+#  Numenta Platform for Intelligent Computing (NuPIC)
+#  Copyright (C) 2020, Numenta, Inc.  Unless you have an agreement
+#  with Numenta, Inc., for a separate license for this software code, the
+#  following terms and conditions apply:
+#
+#  This program is free software you can redistribute it and/or modify
+#  it under the terms of the GNU Affero Public License version 3 as
+#  published by the Free Software Foundation.
+#
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#  See the GNU Affero Public License for more details.
+#
+#  You should have received a copy of the GNU Affero Public License
+#  along with this program.  If not, see htt"://www.gnu.org/licenses.
+#
+#  http://numenta.org/licenses/
+#
+
+import copy
+import os
+
+import torch
+from torch import nn
+
+from nupic.research.frameworks.dendrites.dendritic_layers import GatingDendriticLayer
+from nupic.research.frameworks.pytorch.datasets import omniglot
+from nupic.research.frameworks.pytorch.models import OMLNetwork
+from nupic.research.frameworks.vernon import MetaContinualLearningExperiment, mixins
+from nupic.research.frameworks.vernon.trainables import SupervisedTrainable
+
+
+class OMLExperiment(mixins.OnlineMetaLearning,
+                    MetaContinualLearningExperiment):
+    pass
+
+
+class DendriticNetwork(nn.Module):
```

Comment:
Can you add a few sentences about this class/network? That may be helpful in the future.

Reply:
Added